docs/src/components/DocsLayout.tsx

```diff
@@ -0,0 +1,227 @@
+                  <li>
+                    <a
+                      href="/docs/what-is-nautilus"
+                      className="text-slate-600 hover:text-slate-800"
+                    >
+                      ► What is Nautilus?
+                    </a>
+                  </li>
+                  <li>
+                    <a
+                      href="/docs/how-it-works"
+                      className="text-slate-600 hover:text-slate-800"
+                    >
+                      ► How It Works
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" className="text-slate-600 hover:text-slate-800">
+                      ► Installation
+                    </a>
+                  </li>
+                </ul>
+              </li>
+
+              <li>
+                <h3 className="text-lg font-bold tracking-tight text-slate-900">
+                  🚀 Content Types
+                </h3>
+
+                <ul role="list" className="pl-3 mt-3 space-y-2">
+                  <li>
+                    <a href="#" className="text-slate-600 hover:text-slate-800">
+                      ► What are content types?
+                    </a>
+                  </li>
+
+                  <li>
+                    <a href="#" className="text-slate-600 hover:text-slate-800">
+                      ► Create and edit content types
+                    </a>
+                  </li>
+                </ul>
+              </li>
+
+              <li>
+                <h3 className="text-lg font-bold tracking-tight text-slate-900">
+                  👾 Content
+                </h3>
+
+                <ul role="list" className="pl-3 mt-3 space-y-2">
+                  <li>
+                    <a href="#" className="text-slate-600 hover:text-slate-800">
+                      ► What kind of content can I create and edit?
+                    </a>
+                  </li>
+
+                  <li>
+                    <a href="#" className="text-slate-600 hover:text-slate-800">
+                      ► Previewing content
+                    </a>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </nav>
+        </div>
+
+        <div className="flex-auto bg-white max-w-2xl min-w-0 px-4 py-10 lg:max-w-none lg:pr-0 lg:pl-8 xl:px-16">
+          <h1 className="text-4xl font-bold tracking-tight mt-8 text-black">
+            {title}
+          </h1>
+          <h3 className="text-xl font-semibold tracking-tight mt-4 mb-8 text-slate-400">
+            {description}
+          </h3>
+          <div
+            className="[&_code]:font-semibold [&_code]:text-black [&_h2]:font-bold [&_h2]:text-black [&_h2]:text-3xl [&_h2]:mt-8 [&_h2]:mb-4 [&_hr]:mb-4 [&_p]:text-slate-900 [&_b]:text-black [&_a]:text-sky-500 [&_a]:font-semibold [&_a]:text-lg [&_li]:list-disc [&_li]:text-slate-900"
+            dangerouslySetInnerHTML={{ __html: content }}
```

Comment:
Lol this looks... precarious

Reply:
I could find a better way to do this at some point, but this is a common solution I've seen people do for blogs